server/controllers/favorites.js

```diff
@@ -117,7 +117,7 @@ export default class Favorites {
  * @param {object} req - HTTP Request
  * @param {object} res - HTTP Response
  * @returns {object} Class instance
- * @memberof Favorite
+ * @memberof Favorites
  */
   getSingleFavorite({ params }, res) {
     const { userId, recipeId } = params;
```

Comment:
Go through all your controller method and ensure you are catching internal server errors. (5XX)

Reply:
Also ensure you using the appropriate status codes. For instance,  I see you are catching the error when an object already exist in the database.